example_test.go

```diff
@@ -3,11 +3,12 @@ package carrow_test
 
 import (
 	"fmt"
+	"testing"
 
 	"github.com/353solutions/carrow"
 )
 
-func Example() {
+func TestExample(t *testing.T) {
```

Comment:
Why the rename? This is a [testable exampe](https://blog.golang.org/examples) (it checks the output matches)

Reply:
Cool didn't know this feature.